user-management-oidc.md

```diff
@@ -0,0 +1,68 @@
+User Management - SSO/OIDC
+==========================
+
+> **Note**: OIDC capabilities are only included with a Kubecost Enterprise Subscription.
+
+## Helm configuration
+
+The OIDC integration in Kubecost is fulfilled via the helm configuration parameters `.Values.oidc` [as shown here](https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/cost-analyzer/values.yaml).
+
+## Supported identity providers
+Please refer to the following references to find out more about how to configure the Helm parameters to suit each OIDC identiy provider integration.
+
+### Gluu
+
+Please refer to [Gluu documentation](https://gluu.org/docs/gluu-server/4.0/admin-guide/openid-connect/).
+
+### Okta
+
+Please refer to [Okta documentation](https://developer.okta.com/docs/reference/api/oidc/#request-parameters).
+
+### Azure
+
+Please refer to [Azure documentation](https://learn.microsoft.com/en-us/azure/active-directory/develop/v2-protocols-oidc#send-the-sign-in-request).
+
+### Google OAuth 2.0
+
+Please refer to [Google OAuth 2.0 documentation](https://developers.google.com/identity/openid-connect/openid-connect#authenticatingtheuser).
+
+### Auth0
+
+Please refer to [Auth0 documentation](https://auth0.com/docs/get-started/authentication-and-authorization-flow/add-login-auth-code-flow).
```

Comment:
A more concise way of formatting this (as we recommend consulting respective docs for each provider) would be to list them as bullet points with the docs embedded as a hyperlink in each name. A line could be added under "##Supported Identity Providers" explaining that each link goes to the providers' docs.